integrated_channels/cornerstone/views.py

```diff
@@ -71,16 +177,6 @@ class CornerstoneCoursesListView(generics.ListAPIView):
             empty result.
 
     """
-    permission_classes = (permissions.IsAuthenticated,)
-
-    OAuth2Authentication = get_oauth2authentication_class()
```

Comment:
potentially stupid question: why did we get rid of all the auth stuff here?

Reply:
to avoid redundancies, the auth stuff was added to a base class above

Reply:
oh somehow I missed the re-adding of it elsewhere. 🤦